config/gitleaks.toml

```diff
@@ -2093,6 +2093,22 @@ keywords = [
     "glsa_",
 ]
 
+[[rules]]
+id = "harness-pat"
+description = "Identified a Harness Personal Access Token (PAT), risking unauthorized access to a Harness account."
+regex = '''pat\.[a-zA-Z0-9]{24}\.[a-zA-Z0-9]{24}\.[a-zA-Z0-9]{24}'''
```

Comment:
did you try this out locally? when I create an access token it has pat.{22 chars}.{24 chars}.{20 chars}

Reply:
Updated via https://github.com/gitleaks/gitleaks/pull/1406/commits/a1d989dc32ac7164395d825b8c0f14d324e0daa2